guides/spin/comparison/index.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: single
+title:  "Spin and Roer + Managed Pipeline Templates"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+## Foreward
+
+This document is meant to describe the current state of managing
+pipeline and application config in Spinnaker 'as is', and describe
+a bit about future threads of work in this area.
+
+## `roer` and Managed Pipeline Templates
+
+### What it is
+
+Managed pipeline templates (MPT) is a capability in Spinnaker to
+formulate pipelines as an extended form of Jinja templates
+to allow for templating via variable bindings as well as
+inheritance. The spec for MPT is [here](https://github.com/spinnaker/dcd-spec/blob/master/PIPELINE_TEMPLATES.md).
+The template rendering is done server side in Orca.
+
+Pipeline templates themselves can be managed through Deck or through
+[`roer`](https://github.com/spinnaker/roer), a small CLI for manipulating
+pipeline templates.
+
+### What it isn't
+
+MPT and `roer` are scoped strictly to make managing Spinnaker pipelines
+easier. It is not a generic solution to make API calls to Spinnaker.
+
+### Where it's going
+
+MPT and `roer` are not going away anytime soon. Plenty of folks
+happily use `roer` and MPT.
+
+That being said, it's not actively being developed -- we are
```

Comment:
Worth going a little further to say that some aspects of MPT, especially the in-server template rendering, have posed more technical challenges than expected? That would hopefully help justify why future approaches may shy away from continuing to support server-side templating.